samples/compiler_plugins/xnnpack/test/xnnpack-legalize.mlir

```diff
@@ -38,23 +56,19 @@
 // CHECK:           }
 // CHECK:           return %[[RESULT]] : tensor<1x?x?xf32>
 
-// CHECK-LABEL:   func.func @multiply2(
-// CHECK:           %{{.*}} = call @xnnpack.multiply2
-func.func @multiply2(%a : tensor<?xf32>, %b : tensor<?xf32>) -> tensor<?xf32> {
-  %c = xnnpack.multiply2 %a, %b : (tensor<?xf32>, tensor<?xf32>) -> tensor<?xf32>
-  func.return %c : tensor<?xf32>
-}
-
-// CHECK-LABEL:   func.func @batch_matrix_multiply(
-// CHECK:           %{{.*}} = call @xnnpack.batch_matrix_multiply
-func.func @batch_matrix_multiply(%a : tensor<?x?x?xf32>, %b : tensor<?x?x?xf32>) -> tensor<?x?x?xf32> {
-  %c = xnnpack.batch_matrix_multiply %a, %b : (tensor<?x?x?xf32>, tensor<?x?x?xf32>) -> tensor<?x?x?xf32>
-  func.return %c : tensor<?x?x?xf32>
-}
-
-// CHECK-LABEL:   func.func @fully_connected(
+// CHECK-LABEL:   func.func @fully_connected_0(
 // CHECK:           %{{.*}} = call @xnnpack.fully_connected_nc_qd8_f32_qc4w
 func.func @fully_connected(%a : tensor<1x?x?xi8>, %b : tensor<?x?xi4>) -> tensor<1x?x?xf32> {
   %c = xnnpack.fully_connected_nc_qd8_f32_qc4w %a, %b transpose_rhs = false : (tensor<1x?x?xi8>, tensor<?x?xi4>) -> tensor<1x?x?xf32>
   func.return %c : tensor<1x?x?xf32>
 }
+
+// -----
+
```

Comment:
Where is the CHECK? :p

Reply:
Still a draft haha just wanted to share the code